porcupine/utils.py

```diff
@@ -249,7 +244,7 @@ def invert_color(color: str, *, black_or_white: bool = False) -> str:
 
     # tkinter uses 16-bit colors, convert them to 8-bit
     r, g, b = (value >> 8 for value in widget.winfo_rgb(color))
-    return "#%02x%02x%02x" % (0xFF - r, 0xFF - g, 0xFF - b)
+    return "#{:02x}{:02x}{:02x}".format(0xFF - r, 0xFF - g, 0xFF - b)
```

Comment:
ewww, but whatever

Reply:
An f-string would probably be better :(.